app/models/batch_context.rb

```diff
@@ -103,6 +103,11 @@ def active_globus_url
     globus_destination.url
   end
 
+  # helper method to check if any associated job_runs have completed accessioning
+  def accessioning_complete?
+    job_runs.present? && job_runs.any?(&:accessioning_complete?)
+  end
+
   private
```

Comment:
this used to be inlined in the cleanup job, but moved it to here to address rubocop concerns about complexity (and it could be useful elsewhere too)